src/Compilers/Helpers/SimpleAnalyzerAssemblyLoader.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Reflection;
+
+namespace Microsoft.CodeAnalysis
+{
+    /// <summary>
+    /// Loads analyzer assemblies from their original locations in the file system.
+    /// Assemblies will only be loaded from the locations specified when the loader
```

Comment:
You are right, I was under the assumption that HostAnalyzerManager uses Assembly.LoadFrom can directly use the simple assembly loader, but it in fact uses Assembly.Load (http://source.roslyn.io/#Microsoft.CodeAnalysis.Features/Diagnostics/HostAnalyzerManager.cs,414) to ensure our IDE analyzer assemblies prefer ngen image load.